django/db/backends/sqlite3/base.py

```diff
@@ -390,6 +394,7 @@ def _sqlite_datetime_parse(dt, tzname):
         return None
     try:
         dt = backend_utils.typecast_timestamp(dt)
+        # Typecast_timestamp is returning time rather than datetime
```

Comment:
Not sure if this comment is correct. In `typecast_timestamp` I see `return typecast_date` (which return `datetime.date`) and `return datetime.datetime`.